projects/questionset-editor-library/src/lib/components/editor/editor.component.ts

```diff
@@ -1130,4 +1135,16 @@ export class EditorComponent implements OnInit, OnDestroy, AfterViewInit {
     return response;
   }
 
+  saveDraftComments(comment) {
+    this.editorService.updateComment(this.collectionId,comment)
```

Comment:
Are we not showing any success toaster message when comment is saved/updated ?

Reply:
For success status, now toaster added.